Sources/mpc-core-kit-swift/core/Web3AuthOptions.swift

```diff
@@ -0,0 +1,24 @@
+import FetchNodeDetails
+import Foundation
+
+public class CoreKitWeb3AuthOptions {
+    public let web3AuthClientId: String
+    public let manualSync: Bool
+    public let web3AuthNetwork: Web3AuthNetwork
+    public let storageKey: String?
+    public let sessionTime: Int?
+    public let disableHashFactor: Bool
+    public let localStorage: ILocalStorage
+    public let overwriteMetadataUrl: String?
```

Comment:
we dont support this in web yet, any specific reason why we want to support custom metadata url

Reply:
We should add  inline comments on this param so that devs wont pass custom urls by mistake. Metadata url cant be changed once users are registered

Reply:
@AyushBherwani1998

Reply:
Done